outputs/jira.go

```diff
@@ -163,21 +169,26 @@ func (ctx *JiraAPI) createClient() (*jira.Client, error) {
 }
 
 func (ctx *JiraAPI) Send(content map[string]string) error {
-	client, err := ctx.createClient()
+	client, err := createClient(ctx)
 	if err != nil {
 		log.Printf("unable to create Jira client: %s", err)
 		return err
 	}
 
 	if ctx.boardType == "scrum" {
-		ctx.fetchSprintId(*client)
+		ctx.fetchSprintId(client)
 	}
 
 	metaProject, err := createMetaProject(client, ctx.ProjectKey)
 	if err != nil {
 		return fmt.Errorf("Failed to create meta project: %w", err)
 	}
 
+	err = createIssueType(ctx, metaProject)
+	if err != nil {
+		return fmt.Errorf("Failed to create issuetype: %w", err)
```

Comment:
error message needs to be changed as well

Reply:
Changed